package.json

```diff
@@ -13,9 +13,9 @@
   "type": "commonjs",
   "types": "lib/index.d.ts",
   "scripts": {
-    "test": "jest --config=jest.config.json",
+    "test": "node --test",
```

Comment:
Coverage should be enabled by default as is already the case with `tap@18` in other repos

Reply:
The CI only runs `npm run test` so currently it doesn't check coverage

Reply:
```suggestion
    "test": "c8 node --test",
```